scripts/repos.txt

```diff
@@ -39,6 +36,5 @@ fxa-bugzilla-mirror
 grunt-l10n-lint
 fxa-activity-metrics
 mozlog
-mozilla-services/fxa-testing-e2e
```

Comment:
can you keep this one or does it throw for you?

Reply:
Pretty sure this one threw, but let me just run it again and double-check.

Reply:
Actually, it's a different error for that one, 404 not found. I wonder if the `mozilla-services` org is messing up the script somewhere. Closing this while I dig in...

Reply:
Github often 404's on a permission error, perhaps you don't have appropriate edit perms on that repo.

Reply:
You're right, I've reinstated it at the end of the list. So the script still fails for me but at least it fails at the end, not part-way through.